src/styles/index.css

```
@@ -20,3 +38,37 @@ header .buttons-become_collab {
 	background-color: white;
 	border-bottom: solid 1px black;
 }
+
+footer {
```

Instead of using a tag selector, use a class selector
```suggestion
.footer {
```